app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    data = command.split()
```

Comment:
Validate command for three elements. Also you can use as:
command, source_filepath, destination_filepath = command.split()